insta/src/env.rs

```diff
@@ -328,7 +328,7 @@ impl ToolConfig {
         self.snapshot_update
     }
 
-    /// Returns the value of glob_fail_fast
+    /// Returns the value of [`Self::glob_fail_fast`]
```

Comment:
I think this is very tautological (all these methods' docstrings here are somewhat tautological now I'm looking at them). But at least without the quote it refers to the setting rather than the method itself; so we should revert this line

Reply:
Okay, what do you think about this doccomment instead?

```suggestion
    /// Returns if the glop should fail fast
    /// 
    /// Snapshot failures within the glob macro appear (unless `glob_fail_fast` is set) at the end of of the execution.
```

Reply:
Yes, if we can make that into sentence that would be great; the new line you added has the useful info!

Reply:
Making this into one sentence sounds fine. What do you think of the following?

```suggestion
    /// Returns whether the glob should fail fast, as snapshot failures within the glob macro will appear only at the end of execution unless `glob_fail_fast` is set.
```